README.md

```diff
@@ -1,66 +1,64 @@
-
-<!-- PROJECT LOGO -->
-
-
-
-<!-- ABOUT THE PROJECT -->
 ## About The Project
 
+OpenShift LightSpeed is an AI powered assistant that runs on OpenShift and provides answers to product questions using backend LLM services.
 
-
-
-### Built With
-
-This project is built using IBM Watson machine learning.
-
-
-
-<!-- GETTING STARTED -->
 ## Getting Started
 
 To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
 
-
 * Python 3.11 or newer
```

Comment:
Nothing to do with config, but I don't think 'newer' is correct here, 3.12 didn't work for me.

Reply:
interesting, i'll go ahead and change it to just 3.11 since i'm in the file already